orchagent/portsorch.cpp

```diff
@@ -1312,8 +1312,10 @@ bool PortsOrch::bake()
 
     // Check the APP_DB port table for warm reboot
     vector<FieldValueTuple> tuples;
-    bool foundPortConfigDone = m_portTable->get("PortConfigDone", tuples);
-    SWSS_LOG_NOTICE("foundPortConfigDone = %d", foundPortConfigDone);
+    string value;
+    bool foundPortConfigDone = m_portTable->hget("PortConfigDone", "count", value);
+    unsigned long portCount = stoul(value);
+    SWSS_LOG_NOTICE("foundPortConfigDone = %d, portCount = %lu", foundPortConfigDone, portCount);
```

Comment:
is foundPortConfigDone already true, what if it is false, in this case can we skip the check in 1334?

Reply:
If foundPortConfigDone == false, then return at line 1331

---
In reply to: [226145134](https://github.com/Azure/sonic-swss/pull/645#discussion_r226145134) [](ancestors = 226145134)

Reply:
It might be helpful if m_portCount which is the port count in ASIC gets logged together here.